Resources/public/sonata-page.back.js

```diff
@@ -498,7 +498,7 @@
                 $loader.show();
 
                 $.ajax({
-                    url:     url,
+                    url:     url + '&' + $.param({'composer': 1}),
```

Comment:
@rande the from is genrated into CRUDController (createAction), i think its more complicated to inject other parameters from the twig.